src/include/miopen/device_name.hpp

```diff
@@ -49,32 +51,26 @@ std::string inline GetDeviceNameFromMap(const std::string& name)
         {"Fiji", "gfx803"},
         {"gfx800", "gfx803"},
         {"gfx802", "gfx803"},
-        {"gfx803", "gfx803"},
```

Comment:
Why do we need to delete these?

Reply:
This one "converts" gfx803 to gfx803. Obviously, this kind of conversion is not necessary.

Reply:
Oh I see.